### ISSUE TITLE
Create issues for testing that we can close when satisifed it works

### ISSUE THREAD

**Issue by TokugawaTFC** [Gwyrdain/apprentices-workshop#39]
As the title says I think it would be helpful if we have some issues created here so we can track the testing of the different features.

I wanted to do some testing today but was not sure what was left to do.

Gwyrdain, could you create some issues for what still needs to be tested? I think this will help focus what remains and we can discuss them in the separate issues.

**Comment by Gwyrdain** [comment]
I understand what you're asking for, just not how to answer it.  It's kindof a I don't know what I don't know thing.

For anything that I think of that doesn't work or doesn't work as it should, I am posting it as an issue, because at this point I'm mostly working off the issues list as my 'things to do.'  Every time that I post that an issue is resolved or new feature goes in, it would be good if folks could go up and poke a prod that thing.  For example the new room view thing... I just realized that I forgot exits: none.

Otherwise, the best testing recommendation I can make is to write a test area, or try to recreate portions of an existing area so that you see problems or impracticalities.  I know I had started trying to input Nati's funny farm and comparing results in the associated formatted block which was very helpful.

Sorry that this is probably an unsatisfactory answer.
